.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: "Unit test"
+on:
+  - "push"
+  - "pull_request"
+
+jobs:
+
+  unit-test-with-pytest:
+    runs-on: "ubuntu-latest"
+    strategy:
+      matrix:
+        python-version:
+          - "3.11"
+          - "3.10"
+          - "3.9"
+          - "3.8"
+        pyqt-dependency:
+          - "PyQt5"
+            # - "PyQt6"
+
+    steps:
+      - uses: "actions/checkout@v3"
+
+      - name: "Set up Python ${{ matrix.python-version }}"
+        uses: "actions/setup-python@v4"
+        with:
+          python-version: "${{ matrix.python-version }}"
+
+      - name: "Install dependencies in pyproject.toml"
+        run: |
+          pip install .
+          pip install pytest pytest-cov ${{ matrix.pyqt-dependency }}
```

Comment:
Maybe it's better to specify a dependency on pyqt/pyside in the build config. Probably for another PR.

Reply:
Except, perhaps, if it can work with multiple versions, since those are *packages* and not *versions of the same package*. Does PyProject allow requiring "one of the following" type specifications?

Reply:
>Does PyProject allow requiring "one of the following" type specifications?

Yes, I think this is what we need! I haven't started digging in to how to do it yet.